src/styles/global.css

```
@@ -871,34 +908,230 @@ section[id] {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 0.25rem;
+    flex-direction: column;
+    transition: background 150ms ease;
+}
+.wf .mobile-menu__trigger span {
+    display: block;
+    width: 1.125rem;
+    height: 1.5px;
+    background: currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)